src/main/kotlin/com/devfloor/untitled/option/domain/OptionRepository.kt

```diff
@@ -0,0 +1,7 @@
+package com.devfloor.untitled.option.domain
+
+import org.springframework.data.jpa.repository.JpaRepository
+
+interface OptionRepository : JpaRepository<Option, Long> {
+    fun findByTypeIn(types: List<OptionType>): List<Option>
```

Comment:
In 조건에 해당하는 것들을 리스트로 뽑아내는 부분이기 때문에 `All`을 붙여주는건 어떨까요?
```suggestion
    fun findAllByTypeIn(types: List<OptionType>): List<Option>
```

Reply:
반영했습니다!